### PR TITLE
gr-satellites: new port

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

GNU Radio decoders for several Amateur satellites

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15 19A582a
Xcode 11.2 11B41

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/gr-satellites/Portfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+patchfiles-append \
+    patch-volk.diff
+
+# use C++11
+compiler.cxx_standard 2011
+
+depends_build-append \
+    port:swig-python \
+    port:pkgconfig \
+    port:cppunit
```
Comment (mf2k) [inline science/gr-satellites/Portfile]: Please sort these alphabetically.

Comment (ra1nb0w) [reply science/gr-satellites/Portfile]: they are already alphabetically sorted, only inversely ;-)
Done. thanks mf2k

**Comment by michaelld** [comment]
Excellent! Let me download & test.

**Comment by ra1nb0w** [comment]
> Examples are written to `$PREFIX/share/gr-satellites/examples/` ... can you change this to `$PREFIX/share/gnuradio/examples/satellites/` ? There's an effort around to start moving GR related things into GR-specific spaces. This is one place to start :)

aa, ok. it is a "news" intention. done

**Comment by michaelld** [comment]
FYI during destroot:
```
:debug:destroot Executing proc-post-org.macports.destroot-destroot-0
:info:destroot xinstall: mkdir /opt/local/var/macports/build/_opt_sources_MacPorts_ports_github_macports_science_gr-satellites/gr-satellites/work/destroot/opt/local/share/gnuradio/examples
:info:destroot xinstall: mkdir /opt/local/var/macports/build/_opt_sources_MacPorts_ports_github_macports_science_gr-satellites/gr-satellites/work/destroot/opt/local/share/gnuradio/examples/satellites
:error:destroot Failed to destroot gr-satellites: error copying "xinstall": no such file or directory
:debug:destroot Error code: NONE
:debug:destroot Backtrace: error copying "xinstall": no such file or directory
:debug:destroot     while executing
:debug:destroot "$post $targetname"
```

**Comment by ra1nb0w** [comment]
Sorry, my fault. I am trying to fix libfec

**Comment by michaelld** [comment]
You can always just do a post-destroot hook to call `install_name_tool` ... simple works & I don't see this port as being updated regularly so it should be quite stable in this form ... as well as simple & quick!

**Comment by ra1nb0w** [comment]
> You can always just do a post-destroot hook to call `install_name_tool` ... simple works & I don't see this port as being updated regularly so it should be quite stable in this form ... as well as simple & quick!

I was too tired; Fixed with `install_name_tool`

**Comment by michaelld** [comment]
Thanks for all the changes! libfec now looks OK, and gr-satellites builds OK. When I try to load gr-satellites, I get the following -- which I know isn't your doing, but wondering if you know how to fix it:
```
% python2.7
Python 2.7.16 (default, Sep  7 2019, 15:05:55)
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import satellites
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/satellites/__init__.py", line 103, in <module>
    from sat_3cat_1_telemetry_parser import sat_3cat_1_telemetry_parser
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/satellites/sat_3cat_1_telemetry_parser.py", line 26, in <module>
    import sat_3cat_1_telemetry
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/satellites/sat_3cat_1_telemetry.py", line 109, in <module>
    'beacon_id' / (BeaconID * set_delta_min_delay),\
TypeError: unsupported operand type(s) for *: 'Mapping' and 'function'
```

**Comment by ra1nb0w** [comment]
Sorry for the delay; this is my first holiday of the year and at the end I get the meeting quite relaxed ;-) Said that, check that `py27-construct` is 2.9

**Comment by michaelld** [comment]
> Sorry for the delay; this is my first holiday of the year and at the end I get the meeting quite relaxed ;-) Said that, check that `py27-construct` is 2.9

I'm using whatever MacPorts has:
```
% port installed py27-construct
The following ports are currently installed:
  py27-construct @2.8.16_0 (active)
```

Looks like the MP version is quite out of date ... latest is 2.9.45 ...

**Comment by michaelld** [comment]
NP on delays ... glad you were on holiday & relaxed! I hope the MP meeting was fun and useful!

**Comment by michaelld** [comment]
ah ... I see you've put in https://github.com/macports/macports-ports/pull/5489 to get py-construct updated ... very nice! Let me try that & see if I can get it moving along too...

**Comment by ra1nb0w** [comment]
Ops, I was sure that it was already upstream.

**Comment by michaelld** [comment]
I just merged the py-construct update. let's see if this port now works!

**Comment by michaelld** [comment]
works! merging ...

**Comment by ra1nb0w** [comment]
thank you

**Pull request merged**